js/runner_test.go

```diff
@@ -2074,6 +2074,80 @@ func TestMinIterationDurationIsCancellable(t *testing.T) {
 	}
 }
 
+// nolint:paralleltest
+func TestForceHTTP1Feature(t *testing.T) {
+	cases := map[string]struct {
+		godebug               string
+		expectedForceH1Result bool
+		protocol              string
+	}{
+		"Force H1 Enabled. Checking for H1": {
+			godebug:               "http2client=0,gctrace=1",
+			expectedForceH1Result: true,
+			protocol:              "HTTP/1.1",
+		},
+		"Force H1 Disabled. Checking for H2": {
+			godebug:               "test=0",
+			expectedForceH1Result: false,
+			protocol:              "HTTP/2.0",
+		},
+	}
+
+	for name, tc := range cases {
+		t.Run(name, func(t *testing.T) {
+			err := os.Setenv("GODEBUG", tc.godebug)
+			require.NoError(t, err)
+			defer func() {
+				err = os.Unsetenv("GODEBUG")
+				require.NoError(t, err)
+			}()
+			assert.Equal(t, tc.expectedForceH1Result, forceHTTP1())
+
+			r1, err := getSimpleRunner(t, "/script.js", fmt.Sprintf(`var k6 = require("k6");
+			var check = k6.check;
+			var fail = k6.fail;
+			var http = require("k6/http");;
+			exports.default = function() {
+				var res = http.get("https://k6.io/");
```

Comment:
Just a tiny nitpick, but I would find it best if we were using `https://test-api.k6.io/` here instead.

Reply:
I agree, I forgot to comment about it, I would do one more step using https://pkg.go.dev/github.com/grafana/k6@v0.27.1/lib/testutils/httpmultibin so we can just use a local network.

Reply:
Greetings @oleiade & @codebien, 

I originally wanted to use the httpmultibin in the test cases as I noticed that other test cases were doing the same. However, when I use the httpmultibin endpoints and `Replacer.Replace(....)`, the requests seem to fail and I get an i/o timeout error (as seen in the screenshot). Any ideas why this may be happening? 

![Screen Shot 2021-11-30 at 9 50 10 AM](https://user-images.githubusercontent.com/54779205/144070378-26eea670-01a5-4868-ae61-ee0b06981175.png)

Reply:
@sjordhani22 I would guess you didn't configure the runner to use the mutlibin. Please post your code so we can diagnose more specifically the reason, but I would guess you didn't [overwrite hosts](https://github.com/grafana/k6/blob/master/js/runner_test.go#L1180) ?

Reply:
@MStoykov Thanks for the help, that was the issue!